src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -5173,6 +5171,21 @@ public String toString() {
 	}
 
 	/**
+	 * Transforms this {@link Flux} in order to generate a target {@link Flux}.
```

Comment:
Method JavaDocs must be in the imperative.
See the previous for `compose()`: `Defer the transformation...`. 
https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style

Although that is for Spring. Reactor is a different story :smile: